### ISSUE TITLE
[RLlib] timesteps per minute and cpu/gpu load logging

### ISSUE THREAD

**Issue by evo11x** [ray-project/ray#25847]
### Description

It would be very helpful to have something like timesteps / min. and cpu/gpu/ram load logging in tensorboard

### Use case
This can help tweaking the environments and algorithms performance which will reduce the training time and the energy bill :)

**Comment by mvindiola1** [comment]
@evo11x Most of the metrics you mentioned are  already logged to tensorboard and I am pretty sure they are also in the csv/json logs too. Can you clarify what you would like to see that is different from those?

**Comment by evo11x** [comment]
You are right, with gputil installed I have ram and gpu usage, and I see that there are also obs, learn ms, I was just looking for steps/min., because I always have to hover the mouse over the graph to see the time it took each training and when the training is restored it gets messed up.

**Comment by scottsun94** [comment]
cc: @gjoliver
